server/plugin.ts

```diff
@@ -50,9 +49,12 @@ export class TraceAnalyticsPlugin
     this.logger.debug('trace_analytics: Setup');
     const router = core.http.createRouter();
 
-    // const sqlClient: ILegacyClusterClient = core.opensearch.legacy.createClient(SQL_CLUSTER, {
-    //   plugins: [sqlPlugin],
-    // });
+    // const sqlClient: ILegacyClusterClient = core.opensearch.legacy.createClient(
```

Comment:
How come this is commented out instead of removed? Is this in anticipation for future integration?

Reply:
yes, trace analytics currently uses DSL, we might switch to sql/ppl in the future